dnf/cli/commands/repoquery.py

```diff
@@ -121,7 +121,7 @@ def set_argparser(parser):
         whatrequiresform.add_argument("--exactdeps", action="store_true",
                                       help=_('check dependencies exactly as given, opposite of --alldeps'))
         parser.add_argument('--deplist', action='store_true', help=_(
-            "list of all dependencies and what packages it provides"))
+            "show a list of all dependencies and what packages provide it"))
```

Comment:
@keszybz Thanks for corrections, I just overlooked the text (too much inspiration from yum)